src/Conda.jl

```diff
@@ -189,6 +189,21 @@ _quiet() = get(ENV, "CI", "false") == "true" ? `-q` : ``
 "Install miniconda if it hasn't been installed yet; _install_conda(true) installs Conda even if it has already been installed."
 function _install_conda(env::Environment, force::Bool=false)
     if force || !isfile(Conda.conda)
+        if Sys.iswindows() && (!isascii(PREFIX) || ' ' ∈ PREFIX)
```

Comment:
Note that spaces in the path are not supported on any operating system, so it might be good to give an error in general in that case.